src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -94,14 +97,25 @@ const MultiFacilityImpactDashboard: React.FC = () => {
   const rtData = getFacilitiesRtDataById(facilitiesState.rtData, facilities);
   const systemType = facilities[0]?.systemType;
   const stateName = facilities[0]?.modelInputs.stateName;
-  const newlyAddedReferenceData =
-    !scenario?.referenceDataObservedAt ||
-    Object.values(facilitiesState.referenceFacilities).some((refFacility) => {
-      return (
-        scenario?.referenceDataObservedAt &&
-        isAfter(refFacility.createdAt, scenario.referenceDataObservedAt)
-      );
-    });
+  const showSyncReferenceFacilitiesBaseConditions =
+    referenceFacilitiesEligible &&
+    !facilitiesState.loading &&
+    !scenarioState.loading &&
+    !readOnlyMode; // i.e. User must own of the Scenario
+
+  const showSyncNoUserFacilities =
+    showSyncReferenceFacilitiesBaseConditions && facilities.length == 0;
+
+  const showSyncNewReferenceData =
```

Comment:
@daschi - Ultimately I was able to get things to be less "jerky" using a combination of your suggestions and also setting the `referenceDataObservedAt` when the `SyncNoUserFacilities` first loads as opposed to when the user clicks "Save" on the last modal screen